package.json

```diff
@@ -72,8 +73,9 @@
     "dev": "rollup -c rollup.config.js --watch",
     "update-icons": "node ./src/scripts/update-icons.js",
     "check": "svelte-check --compiler-warnings \"missing-custom-element-compile-options:ignore\"",
-    "transform-tokens": "node ./src/tokens/transformTokens.js && tsc -p ./tsconfig-tokens.json",
+    "transform-tokens": "node ./src/tokens/transformTokens.js && tsc -p ./tsconfig-tokens.json && npm run transform:icons",
```

Comment:
hmm, these aren't tokens, so I'm wondering if we just leave it as a separate command, but bundle with  `npm run build`. Then there's still a way just to update the tokens.

Reply:
This actually could probably move to an action for Skia instead of being a separate npm script. Cf: https://github.com/brave/leo/tree/main/src/tokens/transformation/tailwind/index.js#L70-L74

Reply:
Moved to it's own script, which get run at `build`